src/util/sandbox.cpp

```diff
@@ -359,6 +359,90 @@ SecurityTokenPointer Sandbox::openTokenFromBookmark(const QString& canonicalPath
     return SecurityTokenPointer();
 }
 
+QString Sandbox::migrateOldSettings() {
```

Comment:
The content of the whole function is only suitable for \__APPLE__. Please extend the inner ifdef.

Reply:
I disagree. That would break syntax highlighting in my IDE for the rest of the function which makes it harder to maintain. That is how the rest of this class is written.

Reply:
I disagree. The hard-coded file system paths in this code are macOS specific and don't make any sense for any other platform.

Reply:
Are we really going to hold back a critical bug fix which makes it difficult for users to test other critical bug fixes for the sake of an inconsequentially smaller binary? I do not have an IDE set up on a Mac with indexing support. The more I can do without specifically needing a Mac, the easier this is to maintain.